src/components/Carousel/Home/PartnersLogoCarousel.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { partnersList } from "../../../utils";
```

Comment:
remove unused variable and remove it from utils as well since it's no longer in use